airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/strategies/cursor_pagination_strategy.py

```diff
@@ -50,7 +50,7 @@ def initial_token(self) -> Optional[Any]:
         return self._initial_cursor
 
     def next_page_token(self, response: requests.Response, last_page_size: int, last_record: Optional[Record]) -> Optional[Any]:
-        decoded_response = self.decoder.decode(response)
+        decoded_response = next(self.decoder.decode(response))
```

Comment:
is there a chance decoder.decode will return an empty iterable? If yes, let's catch the exception and raise a user-friendly error message

Reply:
bump @artem1205